src/postgres-util/src/tunnel.rs

```diff
@@ -265,11 +268,13 @@ impl Config {
                 Ok(client)
             }
             TunnelConfig::AwsPrivatelink { connection_id } => {
-                let (host, port) = self.address()?;
+                // We ensure we set the tls verify host correctly, as well as configuring tcp
+                // connection correctly.
```

Comment:
I feel like this comment could be more valuable! As it stands .. well I'd hope all of our code does things correctly without saying so! IMO, should either describe what correctness means here, or just let it stand without commentary.

Reply:
omg another patch required: https://github.com/MaterializeInc/rust-postgres/pull/19 rip